src/Hop/Location.elm

```diff
@@ -35,8 +35,11 @@ locationToFullPath config location =
         "#/"
       else
         ""
+
+    dedupSlash =
+      Regex.replace Regex.All (Regex.regex "/+") (\_ -> "/")
```

Comment:
this may only want to de-dup the slash if it's the very first match in this line; I'm not sure if there are other instances where multiple slashes would occur where we'd want to retain them, so if there are, I can make this less aggressive. If that's the case, I'd like to get a test case in there covering other places where multiple slashes do in fact parse correctly, so we don't introduce regressions later.